odoo/addons/base/models/ir_fields.py

```diff
@@ -192,6 +193,19 @@ def to_field(self, model, field, fromtype=str):
             return None
         return functools.partial(converter, model, field)
 
+    @api.model
+    def _str_to_json(self, model, field, value):
+        if not value:
+            return "", []
```

Comment:
Why does an empty input result in an empty string? In fact why is this handled at all? It doesn't seem to be under consideration for other types.